Authenticator/Source/SegmentedControlRowCell.swift

```diff
@@ -76,21 +68,20 @@ class SegmentedControlRowCell<ViewModel: SegmentedControlRowModel>: UITableViewC
             let segment = viewModel.segments[i]
             segmentedControl.insertSegmentWithTitle(segment.title, atIndex: i, animated: false)
         }
-        // Store values
-        values = viewModel.segments.map { $0.value }
-        // Select the initial value
-        segmentedControl.selectedSegmentIndex = values.indexOf(viewModel.value) ?? 0
+        // Store the action associated with each segment
+        actions = viewModel.segments.map({ $0.action })
+        // Select the initial segment
+        segmentedControl.selectedSegmentIndex = viewModel.selectedSegmentIndex ?? UISegmentedControlNoSegment
     }
 
-    static func heightWithViewModel(viewModel: ViewModel) -> CGFloat {
+    static func heightWithViewModel<ViewModel: SegmentedControlRowModel where ViewModel.Action == Action>(viewModel: ViewModel) -> CGFloat {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 140 characters (line_length)